Source/Revit.IFC.Export/Revit.IFC.Export.csproj

```diff
@@ -358,10 +365,10 @@
       <Version>4.6.6</Version>
     </PackageReference>
     <PackageReference Include="GeometryGymIFC">
-      <Version>0.1.22</Version>
+      <Version>0.0.20</Version>
     </PackageReference>
```

Comment:
Any reason we are reverting the version of GeomtryGym?  Doesn't matter much, just curious.